File: helloprogram.java
import java.io.File;
import java.util.Scanner;
class helloprogram
{
	public static void main(String args[]) throws Exception
	{
		File file= new File("C:\\Users\\srava\\OneDrive\\Desktop\\input.txt");
		Scanner sc=new Scanner(file);

		while(sc.hasNextLine())
		{
			//System.out.println(sc.nextLine());
			String[] name=sc.nextLine().split(",");
			if(name[0].equals("Last Name") == false)
				System.out.println("Welcome,"+name[1]+" "+name[0]);
		}
	}
}

File: helloprogram.R

# read.csv function will read the data from the csv file into data frame type
data <- read.csv("C:\\Users\\srava\\OneDrive\\Desktop\\input.csv")

i<- 1

#nrow calculates the number of rows
# paste function is used to combine the arguments
while(i< nrow(data))
{
  a<-"Welcome, "
  c <-data[c(i),c(1)]
  b <-data[c(i),c(2)]
  print(paste(a,b,c))
  i=i+1 
}

